### PR TITLE
Hotfix/phase two tidy up

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Closes https://github.com/DemocracyClub/Website/issues/604

### CODE REVIEW

File: democracy_club/urls.py [hunk 1]
```diff
@@ -66,8 +66,8 @@
         name="jobs",
     ),
     path(
-        "about/impact/",
-        TemplateView.as_view(template_name="about/impact.html"),
+        "impact/",
```
Comment (symroe) [inline democracy_club/urls.py:64]: As this is [indexed by Google](https://www.google.com/search?q=inurl%3Aimpact+site%3Ademocracyclub.org.uk&hl=en&ei=sKhFY86zJInRgQa06aiQAg&ved=0ahUKEwjOy4LK2tj6AhWJaMAKHbQ0CiIQ4dUDCA4&uact=5&oq=inurl%3Aimpact+site%3Ademocracyclub.org.uk&gs_lcp=Cgdnd3Mtd2l6EANKBAhBGAFKBAhGGABQ7AdY7Adg5ghoAXAAeACAATeIATeSAQExmAEAoAEBwAEB&sclient=gws-wiz), can you set up a `RedirectView` please?

File: democracy_club/urls.py [hunk 2]
```diff
@@ -27,11 +27,6 @@
         name="auth_logout",
     ),
     path("", HomeView.as_view(), name="home"),
-    path(
-        "data_apis/index/",
-        TemplateView.as_view(template_name="data_apis/index.html"),
-        name="data_and_apis",
-    ),
```
Comment (symroe) [inline democracy_club/urls.py:34]: Fixes #604 ?

I think this was a short term URL, so we might not need a redirect for this one?

File: democracy_club/urls.py [hunk 3]
```diff
@@ -150,6 +145,14 @@
         "about/support-us/",
         RedirectView.as_view(url=reverse_lazy("donate")),
     ),
+    path(
+        "about/impact/",
+        RedirectView.as_view(url=reverse_lazy("impact")),
+    ),
```
Comment (VirginiaDooley) [inline democracy_club/urls.py:151]: @symroe I've added the `RedirectView` here. Did I misunderstand?

File: democracy_club/urls.py (hunk 2 above)
Comment (VirginiaDooley) [reply democracy_club/urls.py:34]: That's correct, it doesn't exist anymore - but wasn't sure if we needed to redirect to address the sentry errors - even if just temporarily. Shall I remove?

File: democracy_club/urls.py (hunk 3 above)
Comment (symroe) [reply democracy_club/urls.py:151]: Sorry, I was reading too quickly! This is fine :+1:

File: democracy_club/urls.py (hunk 2 above)
Comment (symroe) [reply democracy_club/urls.py:34]: I think the sentry error was just from the old view, so this is also ok! :+1:

**Review by symroe — APPROVED** [review]

**Pull request merged**